skills/projectCommand.js

```diff
@@ -0,0 +1,18 @@
+const Project = require('../lib/project.js');
+const AddLinkToProjectAction = require('../lib/actions/addLinkToProjectAction.js');
+const Actions = require('../lib/actions/actions.js');
+const AddLinkToProjectResponse = require('../lib/actions/AddLinkToProjectResponse.js');
+
+
+module.exports = function(controller) {
+  controller.on('slash_command', async function (bot, message) {
+    if(message.command === '/project'){
```

Comment:
![](https://media1.giphy.com/media/mXnO9IiWWarkI/200.gif) 

What you have going on in here is 💯 what we're heading towards!

Reply:
Thanks!